internal/store/persistentvolume.go

```diff
@@ -219,6 +220,14 @@ func persistentVolumeMetricFamilies(allowAnnotationsList, allowLabelsList []stri
 				case p.Spec.PersistentVolumeSource.CSI != nil:
 					csiDriver = p.Spec.PersistentVolumeSource.CSI.Driver
 					csiVolumeHandle = p.Spec.PersistentVolumeSource.CSI.VolumeHandle
+
+					for k, v := range p.Spec.PersistentVolumeSource.CSI.VolumeAttributes {
+						if k == "mapOptions" {
+							csiMapOptions = v
+						} else if k == "mounter" {
+							csiMounter = v
+						}
+					}
```

Comment:
These attributes are CSI drivers specific and are not defined in any CSI spec right?

Reply:
Yes, they are... Is there any way we can make this more generic while avoiding big cardinality increases?

Reply:
wdyt about creating a new metric, not enabled by default, that would contain the unbound list of volume attributes?
and then anyone could make it active through `MetricOptInList`